lib/ts/BufferWriterFormat.cc

```diff
@@ -755,6 +756,165 @@ bwf_register_global(std::string_view name, BWGlobalNameSignature formatter)
   return ts::bw_fmt::BWF_GLOBAL_TABLE.emplace(name, formatter).second;
 }
 
+BufferWriter &
+bwformat(BufferWriter &w, BWFSpec const &spec, bwf::Errno const &e)
+{
+  // Hand rolled, might not be totally compliant everywhere, but probably close enough.
+  // The long string will be locally accurate.
+  // Clang requires the double braces. Why, Turing only knows.
+  static const std::array<std::string_view, 134> SHORT_NAME = {{
```

Comment:
Is there no way to have the size of the array determined by what is provided? It's not terrible, since the list of what errors exist isn't likely to change any time soon, but perhaps I'm missing the advantage of `std::array` over `char const * const SHORT_NAME[]`? You could even do an array of string_views, if that's helpful for performance?

And again, this isn't critical, but I wouldn't want people copying this style in places where the list length is more likely to change.

Reply:
(And now I see the stale reviews github hid from me. Looks like this has already been hashed out. I still like C arrays, though. :P )